app/src/main/java/org/oppia/app/options/StoryTextSizeActivityPresenter.kt

```diff
@@ -18,98 +14,21 @@ class StoryTextSizeActivityPresenter @Inject constructor(
   private var fontSize: String = getStoryTextSize(StoryTextSize.SMALL_TEXT_SIZE)
```

Comment:
Default should be `MEDIUM_TEXT_SIZE `

Reply:
Also, check if you still need this? Because this is mainly used to preserve the configuration change which is needed for an activity but during configuration change fragment should handle this now and not the activity.

Reply:
removed `getStoryTextSize` function because it is no longer needed